proto/s2s.go

```diff
@@ -51,6 +51,22 @@ type Request struct {
 	FinishedJobs int       `json:"finishedJobs"` // the number of finished jobs in the request
 }
 
+// SuspendedJobChain (SJC) represents the data required to reconstruct and resume a
+// running job chain in the Job Runner.
+type SuspendedJobChain struct {
+	// request the SJC is for - unique identifier
+	RequestId string    `json:"requestId"`
+	JobChain  *JobChain `json:"jobChain"`
+	// job.Id -> total number of times a job has been tried
+	TotalJobTries map[string]uint `json:"totalJobTries"`
+	// job.Id -> number of times a job was tried the last time it was run
+	LastRunJobTries map[string]uint `json:"lastRunJobTries"`
```

Comment:
From comments it's unclear why we have 2 job counts. I presume this has to do with sequence retries? I.e. Job has been tried 5 times, and last time (for last seq. retry) it was tried 2 times (which implies something like seq. try 1 + 3 job tries and seq. try 2 + 2 job tries (1 remaining), if it's retry=3).